### PR TITLE
Add intake-cmip to list of drivers

### PR DESCRIPTION
**Pull request by andersy005**

### CODE REVIEW

File: docs/source/plugin-directory.rst [hunk 1]
```diff
@@ -25,6 +25,7 @@ This is a list of known projects which install driver plugins for Intake:
 * `intake-sql <https://github.com/ContinuumIO/intake-sql>`_: Generic SQL queries via SQLAlchemy
 * `intake-splunk <https://github.com/ContinuumIO/intake-splunk>`_: Splunk machine data query
 * `intake-xarray <https://github.com/ContinuumIO/intake-xarray>`_: load netCDF, Zarr and other multi-dimensional data
+* `intake-cmip <https://github.com/NCAR/intake-cmip>`_:  load `CMIP <https://cmip.llnl.gov/>`_ (Coupled Model Intercomparison Project) data
```
Comment (martindurant) [inline docs/source/plugin-directory.rst]: Would you mind putting it in alphabetical order?

Comment (andersy005) [reply docs/source/plugin-directory.rst]: Yes, I can do that.

**Comment by martindurant** [comment]
PS: you can add `[skip ci]` in the commit message to avoid having to wait for Travis to go green for documentation-only PRs.

**Comment by andersy005** [comment]
Thank you the tip! I didn't know you could skip CI by adding `[skip ci]` in the commit message.

**Comment by martindurant** [comment]
Thanks for the change. RTD should rebuild, and your link go live in about 20min.

**Pull request merged**